blob/gcsblob/gcsblob_test.go

```diff
@@ -294,35 +294,50 @@ func TestURLOpenerForParams(t *testing.T) {
 	tests := []struct {
 		name     string
 		query    url.Values
+		prev     Options
 		wantOpts Options
 		wantErr  bool
 	}{
 		{
-			name: "AccessID",
+			name: "Invalid query parameter",
```

Comment:
I've carried these changes forward in #1376. Once that's submitted, let's revert this file.

Reply:
Done.